ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -117,6 +139,9 @@ def create_measurement(
     service_class: str,
     description_url: str,
     directory_out: Optional[str],
+    description: str,
+    collection: str,
+    tags: List[str],
```

Comment:
This isn't a list after all.
```suggestion
    tags: Tuple[str,...],
```

Reply:
I guess you are right we are not expecting any change.

Reply:
I expected Click to pass us a list when we use `default=[], multiple=True`, but apparently it uses a tuple.

https://click.palletsprojects.com/en/8.1.x/options/#multiple-options says the default can be a list or tuple but it doesn't really say that it converts from list to tuple.

Reply:
`Tuple[str]` means a tuple with 1 element which is a `str`.

`Tuple[str,...]` means a tuple with any number of elements, which are `str`s.